pkg/gator/client.go

```diff
@@ -30,4 +30,9 @@ type Client interface {
 
 	// Review runs all Constraints against obj.
 	Review(ctx context.Context, obj interface{}, opts ...client.QueryOpt) (*types.Responses, error)
+
+	// Audit makes sure the cached state of the system satisfies all stored constraints.
+	// On error, the responses return value will still be populated so that
+	// partial results can be analyzed.
+	Audit(ctx context.Context, opts ...client.QueryOpt) (*types.Responses, error)
```

Comment:
does audit make sense for gator since there's no cached state? or maybe I'm missing a scenario.

Reply:
We use Audit to represent when we're evaluating an entire repository. This saves `gator test` from having to replicate the audit logic of iterating over everything.

Reply:
In this case the entirety of the state is cached (see the calls to AddData() added elsewhere)

Reply:
makes sense